magenta/lib/melodies_lib.py

```diff
@@ -248,6 +248,15 @@ def _get_last_on_off_events(self):
         return (i, last_off)
     raise ValueError('No events in the stream')
 
+  def deepcopy(self):
```

Comment:
Done.

On Fri, Sep 2, 2016 at 9:42 AM, cghawthorne <notifications@github.com>
wrote:

> In magenta/lib/melodies_lib.py
> <https://github.com/tensorflow/magenta/pull/193#discussion_r77376380>:
>
> > @@ -248,6 +248,15 @@ def _get_last_on_off_events(self):
> >          return (i, last_off)
> >      raise ValueError('No events in the stream')
> >
> > +  def deepcopy(self):
>
> might as well also define *deecopy* while you're here, in case someone
> wants to use copy.deepcopy
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/tensorflow/magenta/pull/193/files/224dbf3ad637586ea2f5c05d54cd4054245f1c5d#r77376380>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/ABCa6Cs7lAvyslXy-aiKGHK66aYt0X8aks5qmFIIgaJpZM4JzUc5>
> .
>